File: src/jdnow.c
#include <stdio.h>
#include "tpm/astro.h"

/* Simple program used to test astrodate.utc2jd function     */
/* against TPM utc_now function. Use together with jdnow.py. */

int main()
{

  double stamp;
  stamp=utc_now();
  printf("Now (UTC JD): %f\n",stamp);
}

File: lib/astrolib/coords/jdnow.py
""" Simple script used to test astrodate.utc2jd function
against TPM utc_now function. Use together with jdnow.c."""

from astrodate import utc2jd
import datetime

stamp=datetime.datetime.utcnow()
print "Now (GM) = ", stamp

print "Now (UTC JD): %f"%utc2jd(stamp)
